### ISSUE TITLE
Add certification requirements when applying to excursions

### ISSUE THREAD

**Issue by morrigan** [rkgeronimo/wp-web#10]
This is how it should look like for admin (user creating an excursion):
<img width="375" alt="Screenshot 2023-01-29 at 21 59 29" src="https://user-images.githubusercontent.com/4976604/215355408-b0c7b19a-8f98-4360-8902-bcb7632e7bf4.png">

This has been commented out in the frontend file `rkgExcursionMeta.twig` as it didn't have backend implementation (on excursion save and applications check).

**Comment by morrigan** [comment]
> Since "Iskusniji R1+" is not a category, what is considered r1 advanced for this?

Yeah, I am not sure where is this coming from tbh. I don't think it has ever been a problem. Usually information in title/description works for people.

**Comment by mholi21** [comment]
> Yeah, I am not sure where is this coming from tbh. I don't think it has ever been a problem. Usually information in title/description works for people.

I mean this can be easily implemented to check if the user is R2+, or has Nitrox. But the "Iskusniji R1+" is not a selectable category or option in profile so it would not be possible to automatically check it, or a condition for that check would need to be defined.

Should the extra options only be R2+ and Nitrox?

I think this would be good because it would force people to start updating categories and extra things they have. Seems like some don't have that information in the profile, making it sometimes harder for excursion organizer to have full overview of participants.
